apps/libsignpost-mbed/port_signpost_mbed.cpp

```diff
@@ -17,18 +17,21 @@ static DigitalOut ModOut(MOD_OUT);
 static InterruptIn ModIn(MOD_IN);
 static DigitalIn pps(PPS);
 static Serial DBG(SERIAL_TX, SERIAL_RX, 115200);
-static I2C I2Cwriter(I2C_MASTER_SDA, I2C_MASTER_SCL);
-static I2CSlave I2Creader(I2C_SLAVE_SDA, I2C_SLAVE_SCL);
+static I2C* I2Cwriter = NULL;
```

Comment:
does the `I2Cwriter` need to be global? maybe makes sense to just have it in the master_write function below? It doesn't matter now, b/c nothing uses it, but the delete below doesn't reset this pointer to NULL, which could possibly confuse something in the future

Reply:
That's true - I can move that into the function context